Documentation/UpgradeLTS/UpgradeProcess/Index.rst

```diff
@@ -63,4 +63,8 @@ If you were working with "DAM" (Digital Asset Management) in TYPO3
 Abstraction Layer).
 
 DAM and its related extensions no longer work with TYPO3 CMS 6.x.
-For this, read on in the next chapter.
\ No newline at end of file
+For this, read on in the next chapter.
+
+In TYPO3 CMS 4.5 languages could be defined by setting config.language = xy in TypoScript and having according labels in <languageKey index="xy" type="array"> in language files.
```

Comment:
I would suggest to use typewriter for the config part:
... ``config.language = xy`` in TypoScript...

and the same for the XML part